packages/material-ui/src/RadioGroup/RadioGroup.js

```diff
@@ -30,15 +39,22 @@ class RadioGroup extends React.Component {
     focusRadios[0].focus();
   };
 
-  handleRadioChange = (event, checked) => {
```

Comment:
No need to change the name here. Just diff noise and it described the method perfectly before.

Reply:
To compare with the other components, but I believe this change increase consistency.